package.json

```diff
@@ -18,36 +18,41 @@
     "localstorage-down": "^0.6.7",
     "logplease": "^1.2.14",
     "multihashes": "^0.4.12",
+    "orbit-db-access-controllers": "github:orbitdb/orbit-db-access-controllers",
     "orbit-db-cache": "~0.2.4",
-    "orbit-db-counterstore": "~1.4.0",
-    "orbit-db-docstore": "~1.4.3",
-    "orbit-db-eventstore": "~1.4.0",
-    "orbit-db-feedstore": "~1.4.0",
-    "orbit-db-keystore": "~0.1.0",
-    "orbit-db-kvstore": "~1.4.0",
+    "orbit-db-counterstore": "github:orbitdb/orbit-db-counterstore",
+    "orbit-db-docstore": "github:orbitdb/orbit-db-docstore",
+    "orbit-db-eventstore": "github:orbitdb/orbit-db-eventstore",
+    "orbit-db-feedstore": "github:orbitdb/orbit-db-feedstore",
+    "orbit-db-identity-provider": "github:orbitdb/orbit-db-identity-provider",
+    "orbit-db-keystore": "github:orbitdb/orbit-db-keystore",
+    "orbit-db-kvstore": "github:orbitdb/orbit-db-kvstore",
     "orbit-db-pubsub": "~0.5.5",
-    "orbit-db-store": "~2.5.3"
+    "orbit-db-store": "github:orbitdb/orbit-db-store"
   },
   "devDependencies": {
     "babel-core": "^6.26.0",
     "babel-loader": "^7.1.2",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-polyfill": "^6.26.0",
     "babel-preset-es2015": "^6.24.1",
-    "datastore-level": "~0.8.0",
+    "datastore-level": "~0.10.0",
     "go-ipfs-dep": "0.4.13",
-    "ipfs": "~0.30.0",
-    "ipfs-repo": "~0.22.1",
-    "ipfsd-ctl": "^0.37.5",
+    "ipfs": "~0.33.0",
+    "ipfs-repo": "~0.25.0",
+    "ipfsd-ctl": "~0.37.5",
     "markdown-toc": "^1.2.0",
-    "mocha": "^4.0.1",
+    "mocha": "^5.2.0",
     "p-each-series": "^1.0.0",
+    "p-map": "^1.2.0",
     "p-map-series": "^1.0.0",
+    "p-whilst": "^1.0.0",
     "remark-cli": "^5.0.0",
     "remark-validate-links": "^7.0.0",
     "rimraf": "^2.6.2",
-    "uglifyjs-webpack-plugin": "^1.1.4",
-    "webpack": "^3.8.1"
+    "uglifyjs-webpack-plugin": "^2.0.1",
```

Comment:
I believe we can remove Uglify now completely from deps. I can PR the cleaned up code.

Reply:
This is now a PR in https://github.com/orbitdb/orbit-db/pull/528